### PR TITLE
LPS-187531 Method DLUtil.getImagePreviewURL()is now part of the DLURLHelper class

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://issues.liferay.com/browse/LPS-187531

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

File: modules/util/source-formatter/src/main/resources/sourcechecks.xml [hunk 1]
```diff
@@ -1592,6 +1592,10 @@
 			<category name="Upgrade" />
 			<description name="Replaces the references of the method `HtmlUtil.extractText(` with the method `extractText(` of `HtmlParser` class" />
 		</check>
+		<check name="UpgradeJavaGetImagePreviewURLMethodCheck">
+			<category name="Upgrade" />
+			<description name="Replaces the references of the method 'DLUtil.getImagePreviewURL(' with the method 'getImagePreviewURL(' of 'DLURLHelper' class" />
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/resources/sourcechecks.xml:1597]: @TamyrisTorres, no need to include the `(` in `DLUtil.getImagePreviewURL(` and `getImagePreviewURL(`, after this is just documentation, right? :slightly_smiling_face:

**Review by natocesarrego — COMMENTED** [review]
Good job!

File: modules/util/source-formatter/src/main/resources/sourcechecks.xml (hunk 1 above)
Comment (TamyrisTorres) [reply modules/util/source-formatter/src/main/resources/sourcechecks.xml]: The change is done! @natocesarrego

**Review by natocesarrego — APPROVED** [review]

**Comment by natocesarrego** [comment]
Pull request approved, good job, @TamyrisTorres! :slightly_smiling_face:

**Pull request closed without merging**